docs/semgrep-code/policies.md

```diff
@@ -46,10 +46,13 @@ The Policies page consists of a header and three main panes:
         <dd>
             The top header consists of:
             <ul>
-             <li><strong>Policies view drop-down</strong>, which lets you choose between:
-                <ul><li>Grouping rules by vulnerability class</li>
-                <li>No grouping</li>
-                </ul></li>
+             <li>
```

Comment:
![image](https://github.com/semgrep/semgrep-docs/assets/20766840/4d03aacb-d031-4643-8247-8a238e390096)

Render looks a little odd, can we fix please?

Reply:
Sure, fixed.